site/content/en/docs/manual/advanced/automatic-annotation.md

```diff
@@ -2,54 +2,115 @@
 title: 'Automatic annotation'
 linkTitle: 'Automatic annotation'
 weight: 16
-description: 'Guide to using the automatic annotation of tasks.'
+description: 'Automatic annotation of tasks'
 ---
 
-Automatic Annotation is used for creating preliminary annotations.
-To use Automatic Annotation you need a DL model that can be deployed by a CVAT administrator.
-You can find the list of available models in the `Models` section.
+Automatic annotation in CVAT is a tool that you can use
+to automatically pre-annotate your data with pre-trained
+and pre-installed [Models](#models).
 
-1. To launch automatic annotation, you should open the dashboard and find a task which you want to annotate.
-   Then click the `Actions` button and choose option `Automatic Annotation` from the dropdown menu.
+You can also **add models** from popular platforms
+like **Hugging Face** or **Roboflow** to expand the
+range of available options.
+
+> **Note**, that on self-hosted instances the automatic annotation feature is free,
+> but you have to add your own models. Integration with Hugging Face
+> and Roboflow is not supported for self-hosted solution.
+> <br>On CVAT Cloud Automatic annotation is a [paid CVAT feature](https://www.cvat.ai/pricing/cloud)
+> with several pre-installed models available by default,
+> also you can add models from Roboflow and Hugging Face.
```

Comment:
Maybe you could replace this with a table contrasting self-hosted vs cloud? Seems like it would be a bit easier to grasp.

Reply:
Sure!